iconservice/database/db.py

```diff
@@ -240,9 +243,12 @@ def get_from_batch(self,
     def _check_tx_batch_value(context: Optional['IconScoreContext'],
                               key: bytes,
                               include_root_hash: bool):
+        if context.tx_batch.get(key) is None:
```

Comment:
Assign the return value of context.tx_batch.get(key) to another local variable like "value" and use it in the below codes.